internal/Position.go

```diff
@@ -0,0 +1,14 @@
+package internal
+
+type Position interface{}
+
+type PositionRequest struct{
+	ID     string  `json:"id"`
```

Comment:
I don't see the reason to set the ID from outside. It should be generated by the server and being placed to the response. .

Reply:
+